client/src/main/java/org/eclipse/hono/client/impl/CredentialsClientImpl.java

```diff
@@ -96,7 +96,7 @@ public static String getTargetAddress(final String tenantId) {
      * @param creationHandler The handler to invoke with the outcome of the creation attempt.
      * @throws NullPointerException if any of the parameters is {@code null}.
      */
-    public static void create(
+    public final static void create(
```

Comment:
please change the order of the modifiers to `public static final`

Reply:
ok